code/script.js

```diff
@@ -220,106 +226,154 @@ const generateBoard = () => {
       </div>
     `
   })
-}
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+};
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+
+  generateBoard() //Load all the characters
+  setSecret() //Generates a new secret person to guess on at the start
+};
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-
   // This variable stores what option group (category) the question belongs to.
```

Comment:
You've added good comments that explain the code. Can otherwise be really hard to follow what the different variables represent.